examples/notebooks/Splitters Tutorial.ipynb

```diff
@@ -0,0 +1,370 @@
+{
+ "cells": [
```

Comment:
Can you rename without a space and with an _ instead?  The scripts to turn into html/post on the website don't work on filepaths with whitesapce.